### DISCUSSION TITLE
Download latest version of antismash db

### DISCUSSION THREAD

**Discussion by AmayAgrawal** [antismash/antismash#703] [Q&A]
Hi,

Is there any way to download the .gbk files of all BGCs present in the latest antismash database (version 4) that is used for clusterblast?

I already performed the clusterblast locally, but now I need the .gbk files of the clusterblast hits to create phylogenetic tree visualizations

**Comment by zdk123** [comment]
You can find the antismash output for a specific NCBI accession by querying for it here
https://antismash-db.secondarymetabolites.org/query

**Reply by AmayAgrawal** [reply]
Thanks for the suggestion!

I am aware of this and I already tried this solution, but the issue is that I have large number of query BGCs, with each query BGC having handful of hits from clusterblast. So downloading each of the genomes one by one might be very time consuming. That's why I am looking for a quicker way to do this and wanted to download all BGCs together

**Reply by kblin** [reply]
Are we talking about the full genome genbanks, or the cluster gbks? I see both of them are missing from our download server at the moment, but I can fix that.

**Reply by AmayAgrawal** [reply]
I would be interested in the cluster gbks as I got the hits for the specific clusters.

Looking at the statistics on antismash-db website, I see that in total there are 232018 clusters present in the database. It would be nice to download the gbks of all these clusters, so that I can build local database out of it as well. 

It would be nice if you can make all these gbks available on the download server soon

**Reply by AmayAgrawal** [reply]
Hi @kblin,

As mentioned earlier, did you get a chance to look at making all the cluster gbks available on the download server of antismash-db?
